config/webpack/webpack.config.js

```diff
@@ -28,7 +28,10 @@ const styleLoaders = [
       },
   {
     loader: 'css-loader',
-    options: { importLoaders: 1 },
+    options: {
+      importLoaders: 1,
+      sourceMap: isDevelopment,
```

Comment:
Not entirely sure if this is needed: https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#500-2020-10-13

Based on 4.x changelog it should pick `devtool` option but then in 5.x it sounds like it needs to be explicit. 🤔